metadata_service/proxy/neo4j_proxy.py

```diff
@@ -699,16 +699,18 @@ def get_user_detail(self, *, user_id: str) -> Union[UserEntity, None]:
 
         query = textwrap.dedent("""
         MATCH (user:User {key: $user_id})
-        OPTIONAL MATCH (user)-[:manage_by]->(manager:User)
+        OPTIONAL MATCH (user)-[:MANAGE_BY]->(manager:User)
         RETURN user as user_record, manager as manager_record
         """)
 
         record = self._execute_cypher_query(statement=query,
                                             param_dict={'user_id': user_id})
-        if not record:
+        single_result = record.single()
+
+        if not single_result:
```

Comment:
could you add a test to check the user not exist case? Just test the exception will be throw is fine? Thanks.